src/main/java/io/litmuschaos/request/ListProjectRequest.java

```diff
@@ -0,0 +1,63 @@
+package io.litmuschaos.request;
+
+public class ListProjectRequest {
+    private final Integer page;
+    private final Integer limit;
+    private final String sortField;
+    private final Boolean createdByMe;
+
+    private ListProjectRequest(Builder builder) {
+        this.page = builder.page;
+        this.limit = builder.limit;
+        this.sortField = builder.sortField;
+        this.createdByMe = builder.createdByMe;
+    }
+
+    public static class Builder {
+        private Integer page;
+        private Integer limit;
+        private String sortField;
+        private Boolean createdByMe;
```

Comment:
Hi, @suyeon-jung-dev 

Now, we can create ListProjectRequest object like this,
```java
ListProjectRequest request = new ListProjectRequest.Builder().page(1).limit(1).build();
```
I think creating Builder object by `new` keyword is a little bit unnatural. 

How about add static builder() method that return Builder class?
```java
public static Builder builder(){
   return new Builder();
}
```
Then, we can create ListProjectRequest like this,
```java
ListProjectRequest request = ListProjectRequest.builder().page(10).limit(10).build();
```

Reply:
That make sense. I fixed it in this commit [40ea086](https://github.com/litmuschaos/litmus-java-sdk/pull/24/commits/40ea0868ab7f04b0e70ec0aaa06aaa7447e84e06).